rwatch/ui/layer/menu_layer.c

```diff
@@ -384,6 +386,19 @@ static void menu_layer_update_proc(Layer *layer, GContext *nGContext)
 {
     MenuLayer *menu_layer = (MenuLayer *) layer->container;
     GRect frame = layer_get_frame(layer);
+    
+#ifndef PBL_RECT
+    if (!menu_layer->callbacks.draw_background)
+    {
+        graphics_context_set_fill_color(nGContext, menu_layer->bg_hi_color);
+        GPoint scroll_offset = scroll_layer_get_content_offset(menu_layer->scroll_layer);
+        GRect background_rect = GRect(0, 
+                              (layer->frame.size.h / 2) - (MENU_CELL_BASIC_CELL_HEIGHT / 2) - scroll_offset.y, 
```

Comment:
This formula only applies to menus in centered mode, otherwise the cursor is not drawn at the right position